### DISCUSSION TITLE
Status Update

### DISCUSSION THREAD

**Discussion by shawnHartsell** [actions/actions-sync#85] [General]
👋 Hello! My name is @shawnHartsell, a member of the Actions engineering team at GitHub. I wanted to make a quick post regarding the state of this repo as well as our plans going forward.

First of all, I wanted to thank everyone for their contributions so far.  Our team has picked up support for this repo after some internal organization changes.

I am a member of the team that will be maintaining this project going forward. At the moment, the rest of the team is in the progress on getting up to speed on the codebase. Our first action items will be housekeeping duties such as upgrading Go, upgrading dependencies etc. We will also be closing out stale issues and PRs.

Going forward we will primarily focus on maintenance duties such as bug fixes, keeping dependencies up to date, etc. That being said, we still appreciate contributions from the community!

Thanks, and looking forward to working with you all!

**Comment by AdrianDsg** [comment]
Hi @shawnHartsell,
I was wondering if you could share a little status update any time soon. I haven't noticed much public activity since the merge of the dependency upgrade. I'd be very grateful if you would consider sharing a few insights on GitHub's plans for this repo.

**Reply by shawnHartsell** [reply]
@AdrianDsg nothing has changed in the time since I originally authored this post. We are still committed to supporting this project by way of:

- Resolving/Fixing bugs as reported by the community
- Keeping dependencies up to date (dependabot, etc)
- General maintenance work (project set up, updating test coverage, etc)

As far as new features, there is nothing planned...but are open to taking input from the community if there are missing gaps in functionality.
